Testing/SystemTests/tests/analysis/reference/ILL_IN4_SofQW.nxs.md5

```diff
@@ -1 +1 @@
-23f425fa47b9f5ae9820ffec9b00608f
+b740b58fdb96a04595a4068174cbc592
```

Comment:
I could not detect the difference between the old and new files. Are they any different?

Reply:
I've just checked and yes they are different. Here's a screenshot:
![ILL_IN4_SofWQ_differences](https://user-images.githubusercontent.com/56295817/73950626-20ef5f80-48f4-11ea-8948-b0cd050eea87.PNG)
I would expect them to be different because of the fact there's no interpolation happening between the different wavelengths. Even the points that weren't interpolated will also change because of the reversal of the event\wavelength loops in MCAbsorptionStrategy which will change the random number sequence used